src/main/java/seedu/address/ui/TaskCard.java

```diff
@@ -0,0 +1,68 @@
+package seedu.address.ui;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.Region;
+import seedu.address.model.assignment.Task;
+
+/**
+ * An UI component that displays information of a {@code Assignment}.
+ */
+public class TaskCard extends UiPart<Region> {
+
```

Comment:
JavaDocs still refer to Assignment instead of Task

Reply:
amended!